lib/src/main/kotlin/net/dean/jraw/references/PublicContributionReference.kt

```diff
@@ -202,4 +203,62 @@ abstract class PublicContributionReference internal constructor(reddit: RedditCl
                 ))
         }
     }
+
+    /**
+     * Sets the [spoilerStatus] of a post using the official Reddit spoiler flair.
+     *
+     * TODO - Requires mod privileges for the subreddit that contains the post.
+     */
+    @EndpointImplementation(Endpoint.POST_SPOILER, Endpoint.POST_UNSPOILER)
+    fun setPostSpoiler(spoilerStatus: Boolean) {
```

Comment:
Not a huge fan of the method name, the "set" prefix makes it appear like a normal setter and not as an API call. Maybe something like `makeSpoiler` or `flagAsSpoiler`?